main.go

```diff
@@ -220,40 +220,31 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 		Scheme: scheme,
 	}
 
-	config := config.Configuration{}
+	configuration := config.Configuration{}
 
 	if configFile == "" {
-		scheme.Default(&config)
+		scheme.Default(&configuration)
+		options = ctrl.Options{
+			Port:                   config.DefaultWebhookPort,
+			HealthProbeBindAddress: config.DefaultHealthProbeBindAddress,
+			MetricsBindAddress:     config.DefaultMetricsBindAddress,
+			LeaderElectionID:       config.DefaultLeaderElectionID,
```

Comment:
Just created a PR here https://github.com/kubernetes-sigs/kueue/pull/413, if leaderElection is false, leaderElectionID should be empty since we have no config file here.

Reply:
It makes sense.
I removed this leaderElectionID.